cdap-api/src/main/java/io/cdap/cdap/api/app/ApplicationSpecification.java

```diff
@@ -50,6 +50,12 @@
    */
   String getAppVersion();
 
+  /**
+   * @return CDAP version that was used to create or update pipeline for the Application
```

Comment:
would be good to document when it can be null

Reply:
Done